nshimiyejayd/my-brand
# #gMKQ5QOF Implement skills page

#### What does this PR do?
This PR implements Skills page

#### Description of Task to be completed?
- implement Skills page

#### How should this be manually tested?
- Clone the [repo](https://github.com/nshimiyejayd/my-brand)
-  Checkout branch `feat-implement-skills-page-gMKQ5QOF`
-  Open skills.html in your browser to check Skills page
#### What are the relevant pivotal tracker stories?
 [#gMKQ5QOF](https://trello.com/c/gMKQ5QOF/20-skills-page)
